pkg/config/config.go

```diff
@@ -406,6 +410,14 @@ type SocketSlack struct {
 	AppToken string                                 `yaml:"appToken,omitempty"`
 }
 
+type CloudSlack struct {
+	Enabled  bool                                   `yaml:"enabled"`
+	Channels IdentifiableMap[ChannelBindingsByName] `yaml:"channels"  validate:"required_if=Enabled true,dive,omitempty,min=1"`
+	Token    string                                 `yaml:"token"`
+	BotID    string                                 `yaml:"botID,omitempty"`
+	Server   string                                 `yaml:"server"`
```

Comment:
IMO it would be good to already change it from `Server string` to `Server GRPCServer` where the struct is extendable for the next steps (auth, etc.)
```
type GRPCServer struct {
   URL string
  // other fields
}